models/Import/Service/ResultImporter.php

```diff
@@ -71,31 +72,40 @@ function () use ($resultStorage, $input, $testUri, $deliveryExecutionUri, $testS
                 if (null === $testScoreVariables) {
                     return;
                 }
-
                 $this->updateTestVariables(
                     $resultStorage,
                     $testScoreVariables['scoreTotalVariable'],
                     $testScoreVariables['scoreTotalVariableId'],
                     $deliveryExecutionUri,
                     $testUri,
-                    $totalScoreCalculatedByItemOutcomes
+                    $totalScoreCalculatedByItemOutcomes,
+                    $isOutcomeWithNewZeroValue
                 );
             }
         );
     }
 
     /**
-     * @throws ImportResultException
+     * @param AbstractRdsResultStorage $resultStorage
```

Comment:
we do not use phpdoc annotations for parameters where it does give you more information than you can get from parameters list.